### PR TITLE
Devise login

### PR DESCRIPTION
**Pull request by huaphuoc**
add bootstrap and show list posts.

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>TrainingRor</title>
+    <%= csrf_meta_tags %>
+
+    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
```
Comment (tran-hai-mulodo) [inline app/views/layouts/application.html.erb]: @huaphuoc 
At this time, we don't have application.js file. So, this layout maybe crashes. Please remove this line or add application.js file if you use it. Thanks.

Comment (huaphuoc) [reply app/views/layouts/application.html.erb]: Thanks, I will add file application.js.

**Comment by tran-hai-mulodo** [comment]
@huaphuoc 
Maybe you're missing "config/initializers/devise.rb" file for this PR. Please make sure the website can work well when deploy. Thanks.

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (huaphuoc) [reply app/views/layouts/application.html.erb]: I finish add file application.js.

**Pull request merged**